spec/support/capybara.rb

```diff
@@ -9,4 +10,6 @@
   Capybara.register_driver(:poltergeist) do |app|
     Capybara::Poltergeist::Driver.new app, js_errors: true, timeout: 60
   end
+
+  Capybara.save_and_open_page_path = ENV['CIRCLE_ARTIFACTS'] if ENV['CIRCLE_ARTIFACTS']
```

Comment:
@futhr nice catch, it shouldn't be in this repo or any other using Travis. Too little coffee, too many gems to fix ;)